source/common/http/http1/codec_impl.cc

```diff
@@ -912,11 +907,11 @@ void ConnectionImpl::dumpState(std::ostream& os, int indent_level) const {
 
 void ServerConnectionImpl::dumpAdditionalState(std::ostream& os, int indent_level) const {
   const char* spaces = spacesForLevel(indent_level);
-  os << DUMP_MEMBER(active_request_.request_url_,
-                    active_request_.has_value() &&
-                            !active_request_.value().request_url_.getStringView().empty()
-                        ? active_request_.value().request_url_.getStringView()
-                        : "null\n");
+  os << DUMP_MEMBER_AS(active_request_.request_url_,
+                       active_request_.has_value() &&
+                               !active_request_.value().request_url_.getStringView().empty()
+                           ? active_request_.value().request_url_.getStringView()
+                           : "null\n");
```

Comment:
there's a trailing newline after "null", but no similar newline is added when the URL is not empty.

Reply:
Done, they both have the newline now.